docs/development_testing.md

```diff
@@ -18,7 +18,9 @@ It might be required that you install additional packages depending on your dist
 - Fedora: `sudo dnf -y install systemd-devel gcc-c++`
 - Ubuntu: `sudo apt-get install libudev-dev`
 
-**Important:** Run `tox` before you create your pull request to avoid annoying fixes.
+**Important:** Run `tox` before you create your pull request to avoid annoying fixes. 
+
+**Note:** Running the full `tox` test suite will take quite some time, so as the minimal requirement for pull requests, run at least the tests that are related to your code changes (see details below on how to). The full test suite will anyway be run by the CI once you created your pull request and before it can be merged.
```

Comment:
```suggestion
:::info Important
Run `tox` before you create your pull request to avoid annoying fixes. 
:::

:::note
Running the full `tox` test suite will take quite some time, so as the minimal requirement for pull requests, run at least the tests that are related to your code changes (see details below on how to). The full test suite will anyway be run by the CI once you created your pull request and before it can be merged.
:::
```

We probably should use the admonitions to be consistent with the rest of the site and for for visual punch.

![image](https://user-images.githubusercontent.com/12411302/104291430-6d073480-54b3-11eb-9d17-6722d9cf1dc1.png)